_posts/2018-07-25-the_incredibles_3.md

```diff
@@ -0,0 +1,254 @@
+entirely separated from her family.
+
+So, in her new costume, Violet goes out and finds Khaled. Khaled shows her his
+evidence so far and they team up to get the last bit of evidence they need to
+prove and defang the conspiracy. When they get there, of course, Peter also
+arrives and they have it out verbally and then physically.
+
+In here there's a crucial moment where Khaled suggests to Violet that she do
+something she doesn't feel comfortable with; something that Peter had previously
+gotten her to do. She declines and he responds with something like, "Oh. Of
+course. You're right," and they figure out another way to achieve their goals.
+Eventually they overcome Peter, despite his strength, get the evidence and, you
+know, win the day or whatever. Importantly, Violet and Khaled do not begin a
+romantic relationship; just a friendship and a partnership.
+
+
+## Themes
+
+So one major theme is how girls and boys are socialized to think about their
+roles as men and women from, well, a girl's point of view. I'd ideally want
+audiences younger than Violet to see this movie and become aware of the toxic
+behavior pattern Peter exhibits and learn to recognize and avoid people who do
+it or not to do it themselves. Never having been an adolescent girl my own self,
+if this ever got to a stage to be fleshed out more it would obviously benefit
+from a genderally… diverse … poly-gendered…?  Genderifically diverse? A writing
+team diverse in gender, anyway.
+
+Another theme is Khaled's being a person of color. I didn't call it out
+explicitly above because I'd kind of expect it to be conveyed most strongly
+visually. Historically, the cast of The Incredibles movies has been
+overwhelmingly white. I wouldn't change that here, but rather than let is be a
+by-product of a white-dominated industry, I want to make a point with it. Part
+of Khaled's social struggles are because there aren't many people in his school
+like him. This also contributes to Violet's and Peter's reading of him as a
+villain initially (even though they don't know it's Khaled specifically). As
+with the previous theme, a racially diverse writing team would be important if
+this were to be further developed.
+
+A third theme is about how anything that seems too good to be true probably is.
+I didn't put this in here on my own, but it's a major factor in the plots of the
+first two films, so it felt appropriate to have an ally-to-villain reversal like
+the first two films and permute it by also having a villain-to-ally reversal.
+
+Lastly, there's the point about what is just and what is legal and how we should
+interrogate our system of laws to try to always make them more just. I am 0%
+interested in Violet and Khaled at the end of the movie working with the cops or
+anything. They should give their evidence to a journalist. Peter is definitely
+the type to work with the cops. Maybe his dad even _is_ a cop.
+
+
```

Comment:
It feels like there was an implicit theme that was exploring the idea that maybe it is OK to do things that are socially wrong sometimes for the "greater good".  Maybe it was just a misreading of the plot.